examples/inverse/plot_psf_ctf_label_leakage.py

```diff
@@ -0,0 +1,209 @@
+#
+# Evaluate leakage based on label-to-label PSF correlations
+# ---------------------------------------------------------
+#
+# Note that correlations ignore the overall amplitude of PSFs, i.e. they do
+# not show which region will potentially be the bigger "leaker".
+
+# get PSFs from Source Estimate objects into matrix
+psfs_mat = np.zeros([n_labels, n_verts])
+
+# Leakage matrix for MNE, get first principal component per label
+for [i, s] in enumerate(stcs_psf_mne):
+    psfs_mat[i, :] = s.data[:, 0]
+
+# Compute label-to-label leakage as Pearson correlation of PSFs
+leakage_mne = np.corrcoef(psfs_mat)
+# Sign of correlation is arbitrary, so take absolute values
+leakage_mne = np.abs(leakage_mne)
+
+# Compute label-to-label leakage as Pearson correlation of PSFs
+leakage_lor = np.corrcoef(psfs_mat)
+# Sign of correlation is arbitrary, so take absolute values
+leakage_lor = np.abs(leakage_lor)
+
+# Save the plot order and create a circular layout
+node_order = list()
+node_order.extend(lh_labels[::-1])  # reverse the order
+node_order.extend(rh_labels)
+
+node_angles = circular_layout(label_names, node_order, start_pos=90,
+                              group_boundaries=[0, len(label_names) / 2])
+
+# Plot the graph using node colors from the FreeSurfer parcellation. We only
+# show the 200 strongest connections.
+fig = plt.figure(num=None, figsize=(8, 8), facecolor='black')
+
+plot_connectivity_circle(leakage_mne, label_names, n_lines=200,
+                         node_angles=node_angles, node_colors=label_colors,
+                         title='MNE Leakage', fig=fig)
+
+###############################################################################
+# Most leakage occurs for neighbouring regions, but also for deeper regions
+# across hemispheres.
+#
+# Save the figure (optional)
+# --------------------------
+#
+# By default matplotlib does not save using the facecolor, even though this was
+# set when the figure was generated. If not set via savefig, the labels, title,
+# and legend will be cut off from the output png file::
+#
+#     >>> fname_fig = data_path + '/MEG/sample/plot_label_leakage.png'
+#     >>> fig.savefig(fname_fig, facecolor='black')
+#
+# Plot PSFs for individual labels
+# -------------------------------
+#
+# Let us confirm for left and right lateral occipital lobes that there is
+# indeed no leakage between them, as indicated by the correlation graph.
+# We can plot the summary PSFs for both labels to examine the spatial extent of
+# their leakage.
+
+# left and right lateral occipital
+idx = [22, 23]
+stc_lh = stcs_psf_mne[idx[0]]
+stc_rh = stcs_psf_mne[idx[1]]
+
+# Maximum for scaling across plots
+max_val = np.max([stc_lh.data, stc_rh.data])
+
+###############################################################################
+# Point-spread function for the lateral occipital label in the left hemisphere
+
+brain_lh = stc_lh.plot(subjects_dir=subjects_dir, subject='sample',
```

Comment:
is there a way to disable/hide the various pyvista sliders when generating figures that aren't going to be interactive? I think that would be helpful here.  I thought there was a way, but looking through the stc.plot docstring I'm not finding it. @GuillaumeFavelier @larsoner

Reply:
I'm also not sure how much is added by the `show_traces` panel here?  The brains tell a pretty convincing story.

Reply:
I'm afraid I'm still stuck with mayavi.

Reply:
ok, but the public documentation gets rendered with pyvista ([here is the version I reviewed](https://23151-1301584-gh.circle-artifacts.com/0/dev/auto_examples/inverse/plot_psf_ctf_label_leakage.html#plot-psfs-for-individual-labels)).  So we may still want/need to tailor the pyvista output to be pedagogically optimal.

Reply:
You can just pass `time_viewer=False` to each of these calls